tmt/cli.py

```diff
@@ -835,3 +835,43 @@ def init(context, path, template, force, **kwargs):
         tmt.Test.create('/tests/example', 'shell', tree, force)
         tmt.Plan.create('/plans/example', 'full', tree, force)
         tmt.Story.create('/stories/example', 'full', tree, force)
+
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#  Status
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+@main.command()
+@click.pass_context
+@click.argument('path', default=tmt.utils.WORKDIR_ROOT)
+@click.option(
+    '-i', '--id', 'id_',help=
+    'Run id (name or directory path) to show status of.', metavar="ID")
+@click.option(
+    '--abandoned', is_flag=True, default=False,
+    help='List runs which have provision step completed but finish step '
+         'not yet done.')
+@click.option(
+    '--active', is_flag=True, default=False,
+    help='List runs where at least one of the enabled steps has not '
+         'been finished.')
+@click.option(
+    '--finished', is_flag=True, default=False,
+    help='List all runs which have all enabled steps completed.')
+@verbose_debug_quiet
+def status(context, id_, path, abandoned, active, finished, **kwargs):
+    """
+    Show status of runs.
+
+    Lists past runs in the given directory filtered using options.
+    /var/tmp/tmt is used by default.
```

Comment:
It would be good to mention what exactly does the `done` status mean. Also at least a short description of the verbose levels would be useful here.

Reply:
I am not completely sure how would I go about describing `done` status. Do you have any suggestions?

Reply:
I meant something like this:

    By default, the last completed step is displayed,
    `done` is used when all enabled steps are completed.